src/app/comments/viewComments/ConfirmationModal.jsx

```diff
@@ -13,7 +13,9 @@ function ConfirmationModal({
 }) {
   return (
     <Modal>
-      <Icon name="exclamation-triangle" size="3x" className="mt-5" />
+      <svg width="61" height="57" viewBox="0 0 61 57" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
> Why are SVG fiiles used in some places but inline SVG in others?
> The separate SVG files seem like better separation of concerns, and they make it clear what each of the icons actually are, so I'd go with them in all cases.
I don't know how to use the reference to the SVG files. Is it with the <Icon> tag?

Reply:
> Now that I think about it, why would the _comment_ pages change at all?

Sorry, I don't understand what you mean

Reply:
Just meant to ask why there are changes under src/app/comments, considering that's just the pages from the comment pilots, which don't affect GoGetta anyway.

Reply:
Oh, or if you're referring to the comment about SVGs, I was wondering why in some places you create a separate .svg file, but in other places (like this one) you put <svg> elements inside the pages' jsx files.
Seems to me the separate files are a better solution in all cases.

Reply:
Separate files is always better, but I don't know how to call them.

Reply:
Oh yeah, definitely. So there are actually 2 ways:
1. In most cases, you can just put the svg file in the /public dir, and then wherever you want it shown you can use an `img` element with a `src` attribute pointing to the svg file. For example, see how we use civic_hall_logo.svg in About.jsx.
2. In some cases you may want to treat the svg as a normal React component. To that end, you can import any svg file as you would a component. However, I don't remember what those cases are, in which this would be preferable to just using the first method... 😅 Anyway, as an example you can see how info.svg is imported in Info.jsx.